### PR TITLE
Add support for multiple trackers

### PR DESCRIPTION
**Pull request by DveMac**
- allow multiple trackers to be defined 
- allow events to be selectively sent to any of the trackers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-ga",
-  "version": "2.2.0",
+  "version": "2.3.0",
```
Comment (SimeonC) [inline package.json]: Please don't bump versions in a PR.

**Comment by SimeonC** [comment]
Apart from those two issues it's a great PR, thanks!

**Comment by SimeonC** [comment]
Awesome, thanks. I'm going to aim to get out a version in the next week or so.

**Comment by ebhoren** [comment]
@SimeonC any plans to rolled out this new version to NPM/Yarn soon?
Thanks

**Pull request merged**